plugins/rebalance/api/resp.go

```diff
@@ -0,0 +1,70 @@
+package rebalance
+
+import (
+	"github.com/pborman/uuid"
+)
+
+// Status represents Rebalance Status
+type Status uint64
+
+const (
```

Comment:
Use only the following. Rebalance Cmd will have different codes for Fix layout and regular rebalance. Also these will be used as `rebalanceapi.<Status>` So `GfDefragStatus` prefix can be removed.

    NotStarted
    Started
    Stopped
    Complete
    Failed